src/main/java/org/folio/search/service/reindex/jdbc/MergeInstanceRepository.java

```diff
@@ -81,4 +87,12 @@ public void updateBoundWith(String tenantId, String id, boolean bound) {
     var sql = UPDATE_BOUND_WITH_SQL.formatted(fullTableName);
     jdbcTemplate.update(sql, bound /*? "true" : "false"*/, id);
   }
+
+  @Override
+  public void deleteEntities(List<String> ids, String tenantId) {
```

Comment:
is this ```deleteEntities``` overrides the one from ```MergeRangeRepository```? maybe it's better to have two separate delete methods, one with tenantId and one without?

Reply:
updated